SurfaceDuoSamples/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  ~
+  -->
+
+<resources>
+    <string name="surface_duo_samples_app_name">Faltbare Apps von Surface Duo</string>
+    <string name="surface_duo_samples_app_title">Faltbare App-Beispiele von Surface Duo</string>
+    <string name="hello_to_the_app">Willkommen bei Foldable Samples von Surface Duo!</string>
+
+    <string name="drag_and_drop_info">Drag &amp; Drop - Zeigt, wie Sie ein Layout erstellen, in das Sie Inhalte aus anderen Apps ziehen und ablegen können.</string>
+    <string name="dual_view_info">Doppelte Ansicht - Zeigt, wie ein Layout mit benutzerdefiniertem Verhalten für einen einzelnen Bildschirm oder einen doppelten Bildschirm erstellt wird.</string>
+    <string name="hinge_angle_info">Scharnierwinkel - Zeigt an, wie der Scharnierwinkelsensor angehört und die Winkelwerte verwendet werden.</string>
+    <string name="list_details_info">Listendetails - Zeigt an, wie das Listendetails-Muster in Surface Duo implementiert wird.</string>
+    <string name="pen_events_info">Stiftereignisse - Zeigt an, wie Sie die Stiftereignisse verfolgen und die Metriken verwenden.</string>
```

Comment:
If we use here the German name, we should use it on all resources from this file too.